test/ctia/test_helpers/collections.clj

```diff
@@ -0,0 +1,6 @@
+(ns ctia.test-helpers.collections
+  (:require [clojure.test :refer [is]]))
+
+(defn is-submap? [a b]
```

Comment:
what's with Javanesque name? why not simply `submap?`

Reply:
because it performs a `is` and a submap? :-)
I initially splitted it in 2 funcitons (`submap?` and `is-submap?`) with `(is (submap? a b))` but it breaks the diff rendering in cider. Since it's a test helper, I implemented only this version which actually helps to debug :-)